### ISSUE TITLE
Cost Center Mandate Error Message on Material Transfer of Stock Entry 

### ISSUE THREAD

**Issue by elan-k** [frappe/erpnext#30298]
### Information about bug

**Getting the Following Error While Trying to Submit the Material Transfer of Stock Entry,** 
![image](https://user-images.githubusercontent.com/78781781/158867318-747d8f58-22ad-4acf-9ac6-c02e5ee6b254.png)

**But I could not find the cost centre field on the item details (edit from item grid),** 
![image](https://user-images.githubusercontent.com/78781781/158867531-f5e53d98-5f2b-43e1-bd6e-1eeb846bb745.png)

So it's blocking the user from submitting the transfer record. 

### Steps to Reproduce the Bug
1. Open Stock Entry 
2. Select Material Transfer on Stock Entry Type
3. Provide Required Source & Target Warehouse
4. Select the available item, qty and respective batch on the Item Grid 
5. Try to submit the Record after saving the same. 
6. Getting the Error like "Stock Entry MAT-STE-2022-02593: Cost Center is mandatory for Item "

### Module

stock

### Version

ERPNext: v13.22.1 (HEAD)
Frappe Framework: v13.22.1 (HEAD)
POS Awesome: v3.2.1 (HEAD)

### Installation method

FrappeCloud

**Comment by ankush** [comment]
https://github.com/frappe/erpnext/blob/8819ba07a9e806d24b6567bec22b55a56f5dc073/erpnext/stock/doctype/stock_entry_detail/stock_entry_detail.json#L330 


The field is there, maybe you've made it hidden.

**Comment by elan-k** [comment]
Hi @ankush, 

Thank you for your quick reply. 

Yes, the field is available on Stock Entry Details DocType and it's been used in different Stock Entry Options (Material Receipt, Material Issue, etc). 

But it is not getting displayed for the Stock Transfer option of Stock Entry. 
![image](https://user-images.githubusercontent.com/78781781/159134810-99deca18-b9f6-4c21-9ab4-61dc81e6e02f.png)

As you mention, I have checked the customize option of that DocType "Stock Entry Detail" in my server (check the option below) 
![Cost Center Issue On Stock Transfer](https://user-images.githubusercontent.com/78781781/159135071-4046a59b-e0c1-4529-a480-b1a125892ba3.gif)

Let me show you the Perpetual Inventory Flag (Enabled) in the Company Settings, 
![image](https://user-images.githubusercontent.com/78781781/159135153-e728a9fc-0496-4dbd-b42a-05b5655f5c70.png)

Also, we have encountered or noticed this issue after updates to v13.22.1 on our production server recent. 

Plz help us to identify and resolve the issue as quickly as possible. 

If possible, plz keep this thread in "OPEN" status.
